specification/voiceservices/data-plane/Microsoft.VoiceServices/stable/2023-10-01/swagger.json

```
@@ -0,0 +1,1015 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.VoiceServices.ACG.ProvisioningPlatform.API",
+    "version": "2023-10-01"
+  },
+  "securityDefinitions": {
+    "OAuth2": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "application",
+      "tokenUrl": "https://login.microsoftonline.com/common/oauth2/v2.0/token",
+      "scopes": {
+        "https://func-voiceservice-rp-prod-eastuseuap.azurewebsites.net/.default": "Grants the associated roles assigned to the client application"
+      }
+    }
+  },
+  "security": [
+    {
+      "OAuth2": [
+        "https://func-voiceservice-rp-prod-eastuseuap.azurewebsites.net/.default"
+      ]
+    }
+  ],
+  "paths": {
+    "/account/{accountName}/numbers": {
+      "get": {
+        "operationId": "Account_GetNumbers",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountName",
+            "pattern": "^([A-Za-z0-9_-]+)$",
+            "minLength": 1,
+            "maxLength": 100,
+            "required": true,
+            "type": "string",
+            "description": "Name of the account. Can only contain letters, numbers, underscores and dashes. Can be up to 100 characters in length."
+          },
+          {
+            "format": "int32",
+            "in": "query",
+            "name": "maxpagesize",
+            "type": "integer",
+            "description": "Maximum number of records to include in a single response."
+          },
+          {
+            "format": "int32",
+            "in": "query",
+            "name": "skip",
+            "type": "integer",
+            "description": "An offset into the collection of the first record to be returned."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/NumberResponseDtoPaginatedListDto"
```

Schema names should be clear and concise.
```suggestion
              "$ref": "#/definitions/NumberList"
```